src/editors/vim.ts

```diff
@@ -1,24 +1,24 @@
-import "./editor";
+import Editor from "./editor";
 import { CommandExists } from "../lib/command-exists";
 
-export default class Vim implements Editor {
+export default class Vim extends Editor {
   private commandExists = new CommandExists();
 
   public get name(): string {
-    return "vim";
-  }
-
-  public get displayName(): string {
     return "Vim";
   }
 
   public get icon(): string {
     return "";
   }
 
+  public get binary(): string {
+    return "vim";
+  }
+
   public async isEditorInstalled(): Promise<boolean> {
     // @ts-ignore
-    return this.commandExists.exists(this.name);
+    return this.commandExists.exists(this.binary);
```

Comment:
Then here we loop over binaries and return true if any of the binaries exist in PATH.

Reply:
done